### ISSUE TITLE
how to disable typechecking?

### ISSUE THREAD

**Issue by windrg** [fannheyward/coc-pyright#1051]
**What's the output of `:CocCommand pyright.version`**
>> coc-pyright 1.1.348 with Pyright 1.1.350

**What's the output of `:CocCommand workspace.showOutput Pyright`**
>> coc-pyright 1.1.348 with Pyright 1.1.350
Workspace: /home/draka/ws/my_github/myproj
Using python from /usr/bin/python

[Info  - 21:54:50.147] Pyright language server 1.1.350 starting
[Info  - 21:54:50.148] Server root directory: file:///home/draka/.config/coc/extensions/node_modules/coc-pyright/node_modules/pyright/dist
[Info  - 21:54:50.154] Starting service instance "myproj"
[Info  - 21:54:50.173] Setting pythonPath for service "myproj": "/usr/bin/python"
[Info  - 21:54:50.177] Loading configuration file at /home/draka/ws/my_github/myproj/pyrightconfig.json
[Info  - 21:54:50.213] Assuming Python version 3.8
[Info  - 21:54:50.300] No source files found.
[Info  - 21:55:09.928] Starting service instance ".vim"
[Info  - 21:55:09.939] Setting pythonPath for service ".vim": "/usr/bin/python"
[Info  - 21:55:09.975] Assuming Python version 3.8
[Info  - 21:55:10.194] Found 9 source files

** What is this message?
![image](https://github.com/fannheyward/coc-pyright/assets/506484/b0039e16-c8a8-4c2f-afb8-44c73ec0d2b4)
I'm a very newbie in python. But I'd like to get rid of this warning or error message anyhow.
I can see it goes away when I added a sort of type hint at the end of that line like this.

` # type : ignore`

But I'd like to suppress such messages in all of my python projects.

** my coc-settings.
```

  "python": {
	"analysis" : {
		  "autoImportCompletion" : true,
		  "autoSearchPaths" : true,
		  "diagnosticMode" : false,
		  "typeCheckingMode" : "off",
		  "useLibraryCodeForTypes" : true,
	  },

	"pythonPath" : "python",
	"venvPath" : "",

	"formatting" : {
	  },


  },

  "pyright": {
	  "enable" : true,
	  "completion" : {
		  "importSupport" : true,
		  "snippetSupport" : true,
	  },
	  "pyright.organiseimports.provider" : "pyright",
	  "inlayHints" : {
		  "functionReturnTypes" : true,
		  "variableTypes" : true,
		  "parameterTypes" : true,
	  },

  },
```

**Comment by fannheyward** [comment]
Enable `pyright.disableDiagnostics`.

**Comment by windrg** [comment]
thanks. it works!!
